api/src/pcapi/workers/bank_information_job.py

```diff
@@ -1,12 +1,8 @@
-from pcapi.infrastructure.worker_container import save_offerer_bank_informations
 from pcapi.infrastructure.worker_container import save_venue_bank_informations
 from pcapi.workers import worker
 from pcapi.workers.decorators import job
 
 
 @job(worker.default_queue)
-def bank_information_job(application_id: str, refferer_type: str, procedure_id: str):  # type: ignore [no-untyped-def]
-    if refferer_type == "offerer":
-        save_offerer_bank_informations.execute(application_id)
-    elif refferer_type == "venue":
-        save_venue_bank_informations.execute(application_id, procedure_id)
+def bank_information_job(application_id: str, procedure_id: str):  # type: ignore [no-untyped-def]
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion (non-bloquant):** ‏On peut en profiter pour corriger le `type: ignore` en ajoutant `-> None`